src/Views/HardwareView.vala

```diff
@@ -267,6 +279,23 @@ public class About.HardwareView : Gtk.Grid {
         }
     }
 
+    private string get_gpu_name (bool is_primary) {
```

Comment:
The return type of this method needs to be marked as nullable with `string?` as returning `null` is a possibility with this method.